Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -417,3 +429,256 @@ QString FormatSize(qint64 size)
   }
   return QStringLiteral("%1 %2").arg(QString::number(num, 'f', 1)).arg(unit);
 }
+
+// Cache
+
+GameFile::Cache GameFile::Cache::instance;
+
+GameFile::Cache& GameFile::Cache::Get()
+{
+  return instance;
+}
+
+GameFile::Cache::Cache()
+    : file(QString::fromStdString(File::GetUserPath(D_CACHE_IDX) + "gamefile.cache"))
```

Comment:
I would prefer the file name `qt_gamelist.cache`, for consistency with `wx_gamelist.cache`.